values/values.cloudsqlproxy.yaml

```diff
@@ -0,0 +1,33 @@
+server:
+  sidecarContainers:
+    - name: cloud-sql-proxy
+      # It is recommended to use the latest version of the Cloud SQL proxy
+      # Make sure to update on a regular schedule!
+      image: gcr.io/cloudsql-docker/gce-proxy:1.17
+      command:
+        - "/cloud_sql_proxy"
+
+        # If connecting from a VPC-native GKE cluster, you can use the
+        # following flag to have the proxy connect over private IP
+        - "-ip_address_types=PRIVATE"
+
+        # Replace DB_PORT with the port the proxy should listen on
+        # Defaults: MySQL: 3306, Postgres: 5432, SQLServer: 1433
+
+        - "-instances=<YOUR_PROJECT>:us-central1:<YOUR-INSTANCE-NAME>=tcp:5432"
+      # [START cloud_sql_proxy_k8s_volume_mount]
```

Comment:
this looks like it's maybe copied from https://github.com/GoogleCloudPlatform/cloudsql-proxy/blob/master/examples/kubernetes/proxy_with_sa_key.yaml or something else in that repo - maybe we can match the style of the other values files here without comments etc? otherwise we'd need an apache 2 notice header and that seems a little distracting for an example file.

Reply:
ok now i'm the slow one. i'll remove the comment and match the style